go/service/usersearch.go

```
@@ -188,33 +197,159 @@ func contactSearch(mctx libkb.MetaContext, arg keybase1.UserSearchArg) (res []ke
 	return res, nil
 }
 
+var nonDigits = regexp.MustCompile("[^\\d]")
+
+func imptofuQueryToAssertion(typ keybase1.ImpTofuSearchType, val string) string {
+	switch typ {
+	case keybase1.ImpTofuSearchType_PHONE:
+		return fmt.Sprintf("%s@phone", nonDigits.ReplaceAllString(val, ""))
+	case keybase1.ImpTofuSearchType_EMAIL:
+		return fmt.Sprintf("[%s]@email", strings.ToLower(val))
+	default:
+		panic("invalid keybase1.ImpTofuSearchType enum value")
```

can we just return an error instead since the caller returns an error?